paket.dependencies

```diff
@@ -5,9 +5,9 @@ group Main
   source https://api.nuget.org/v3/index.json
   storage:none
 
-  nuget FSharp.Core >= 4.7.2
+  nuget FSharp.Core >= 6.0.4
```

Comment:
I would like to avoid this change and require the minimum version F# core that is possible from this library. Is there any need for this?

Reply:
Don't know actually will double check. And change it back if no problems arise.

Reply:
@Zaid-Ajaj If the minimum version of FSharp.Core is 6.0.4 rather than 4.7.2, I think Ply could be removed as a dependency though, right?

Reply:
I have just pushed changes, where I have downgraded the the FSharp.Core version back to 4.7.2 in the paket.dependencies file, and regenerated the lock file.